templates/pkgs/base.json

```
@@ -25,7 +25,8 @@
     "test": "jest",
     "test.prod": "yarn lint && yarn test --coverage --maxWorkers=2",
     "test.update": "yarn test --updateSnapshot",
-    "test.watch": "yarn test --watch"
+    "test.watch": "yarn test --watch",
+    "yalc": "yarn build && (cd dist && yalc push)"
```

Can you add a `yalc.watch`? 
```suggestion
    "yalc": "yarn build && (cd dist && yalc push)"
    "yalc.watch": "nodemon --watch 'src/**/*' --exec 'yarn yalc'"
```